deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/DeviceParser.java

```diff
@@ -480,84 +465,99 @@ public void setLastActivityTime(Date lastActivityTime)
         return cloudToDeviceMessageCount;
     }
 
     /**
-     * Setter for CloudToDeviceMessageCount
+     * Setter for CloudToDeviceMessageCount.
      *
-     * @param cloudToDeviceMessageCount the value to set CloudToDeviceMessageCount to
+     * @param cloudToDeviceMessageCount The value to set CloudToDeviceMessageCount to.
      */
     public void setCloudToDeviceMessageCount(long cloudToDeviceMessageCount)
     {
-        //Codes_SRS_DEVICE_PARSER_34_029: [This method shall set the value of this object's cloudToDeviceMessageCount equal to the provided value.]
         this.cloudToDeviceMessageCount = cloudToDeviceMessageCount;
     }
 
     /**
-     * Getter for ManagedBy
+     * Getter for ManagedBy.
      *
-     * @return The value of ManagedBy
+     * @return The value of ManagedBy.
      */
     public String getManagedBy()
     {
-        //Codes_SRS_DEVICE_PARSER_34_018: [This method shall return the value of this object's Status.]
         return managedBy;
     }
 
     /**
-     * Setter for ManagedBy
-     * @param managedBy the value to set managedBy to
+     * Setter for ManagedBy.
+     *
+     * @param managedBy The value to set ManagedBy to.
      */
     public void setManagedBy(String managedBy)
     {
-        //Codes_SRS_DEVICE_PARSER_34_017: [This method shall set the value of this object's Status equal to the provided value.]
         this.managedBy = managedBy;
     }
 
     /**
-     * Getter for capabilities
+     * Getter for Capabilities.
      *
-     * @return The value of capabilities
+     * @return The value of Capabilities.
      */
     public DeviceCapabilitiesParser getCapabilities()
     {
-        //Codes_SRS_DEVICE_PARSER_34_018: [This method shall return the value of this object's IotEdge.]
         return capabilities;
     }
 
     /**
-     * Setter for capabilities
-     * @param capabilities the value to set capabilities to
+     * Setter for Capabilities.
+     *
+     * @param capabilities The value to set Capabilities to.
      */
     public void setCapabilities(DeviceCapabilitiesParser capabilities)
     {
-        //Codes_SRS_DEVICE_PARSER_34_017: [This method shall set the value of this object's capabilities equal to the provided value.]
         this.capabilities = capabilities;
     }
 
     /**
-     * Get the security scope for this device
-     * @return the security scope for this device
+     * Get the device scope for this device.
+     *
+     * @return The device scope for this device.
      */
     public String getScope()
     {
         return this.scope;
     }
 
     /**
-     * Set the security scope for this device
-     * @param scope the security scope to set
+     * Set the device scope for this device.
+     *
+     * @param scope The device scope to set.
      */
     public void setScope(String scope)
     {
         this.scope = scope;
     }
+
+    /**
+     * Get the parent scopes for this device.
```

Comment:
Wondering if there is any ms doc explaining this feature! If so, adding a link will be useful.

Reply:
Not yet. It hasn't officially shipped yet.